src/__tests__/core/blocksRange.test.ts

```diff
@@ -243,3 +244,20 @@ test('blocksRange: same block hash provided twice returns one block', async () =
   const result = await execQuery(query);
   expect(result).toEqual(expected);
 });
+
+test('blocksRange: web3.eth.getBlock called with 2nd param false value', async () => {
```

Comment:
This test is not actually spying on `web3.eth.getBlock`, right? From what I see, the test checks that `queryContainsTxs` is invoked and returns false, but still the test is not covering `getBlock(..., true/false)` which is the assertion we want to make.

Reply:
Same in the other variants (blocks, block).